gdk/src/wayland/mod.rs

```diff
@@ -1,4 +1,15 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+mod wayland_device;
```

Comment:
Why is all this not in a separate `gdk-wayland` crate?

Reply:
Could be moved there yes

Reply:
Also the same for gtk4 I assume.

Reply:
remove how? it can't be auto-generated as there's no gir file for gdk3-wayland.

Reply:
From gtk crate and create another crate on the side? That's how I understood @sdroege's comment.

Reply:
Ignore me if what I said makes no sense. But ideally keep it consistent between gtk3/gtk4 :)

Reply:
Oh ok. Well then, keeping things here then.

Reply:
Well, what @sdroege suggested is to have those as a separate crate even if it's manually binded instead of a sub-module of gdk. We already reserved the gdkwayland crate on crates.io for this